packages/components/src/components/expandable/expandable.ts

```diff
@@ -232,11 +232,13 @@ export default class SdExpandable extends SolidElement {
         background: linear-gradient(180deg, var(--gradient));
       }
 
+      :host([inverted]) {
+        --gradient-color-start: rgba(0, 53, 142, 0);
```

Comment:
Maybe we could place the rgb value in a css variable to identify that this is the color primary. What do you think?